Makefile

```diff
@@ -636,7 +636,7 @@ run-android-ui-test-spoon:
 
 .PHONY: apackage
 apackage:
-	cd platform/android && ./gradlew --parallel --max-workers=$(JOBS) assemble$(BUILDTYPE)
+	if [ -d $(“platform/android/MapboxGLAndroidSDK/src/main/jniLibs”) ]; then cd platform/android && ./gradlew --parallel --max-workers=$(JOBS) assemble$(BUILDTYPE); fi	
```

Comment:
The existence of this directory is not an indication of whether the build was successful or not. The test would succeed on subsequent builds, even if they failed.